arch/arm/src/armv7-m/arm_exception.S

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * arch/arm/src/armv7-m/gnu/arm_exception.S
+ * arch/arm/src/armv7-m/arm_exception.S
```

Comment:
So this is compatible with IAR as well?

Reply:
> So this is compatible with IAR as well?

No, I think. But IAR doesn't have the equivalent version under arch/arm/src/armv7-m/iar/, I assume that it provide chip/board specific folder.